<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagerWnd</class>
 <widget class="QMainWindow" name="ManagerWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CheckEVD</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>85</y>
      <width>790</width>
      <height>460</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>70</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>ÂºÄÂßãÊó∂Èó¥</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>70</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>ÁªìÊùüÊó∂Èó¥</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="group">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>70</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>ÈÉ®Èó®</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QDateEdit" name="start_time">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="end_time">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>550</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>550</y>
      <width>30</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>550</y>
      <width>30</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>40</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>ÊåâÂßìÂêçÊü•ËØ¢</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>40</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>üîç</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>550</y>
      <width>40</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>ÂØºÂá∫</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>550</y>
      <width>40</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>ÂØºÂÖ•</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>start_time</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>ManagerWnd</receiver>
   <slot>refill()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>end_time</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>ManagerWnd</receiver>
   <slot>refill()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>group</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>ManagerWnd</receiver>
   <slot>refill()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>694</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ManagerWnd</receiver>
   <slot>pre()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>561</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>560</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>ManagerWnd</receiver>
   <slot>next()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>487</x>
     <y>559</y>
    </hint>
    <hint type="destinationlabel">
     <x>544</x>
     <y>555</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>ManagerWnd</receiver>
   <slot>refill()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>688</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>667</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>name</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButton_3</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>614</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>704</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add</sender>
   <signal>clicked()</signal>
   <receiver>ManagerWnd</receiver>
   <slot>add_user()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>779</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>729</x>
     <y>9</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>ManagerWnd</receiver>
   <slot>export_to()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>707</x>
     <y>557</y>
    </hint>
    <hint type="destinationlabel">
     <x>666</x>
     <y>555</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refill()</slot>
  <slot>pre()</slot>
  <slot>next()</slot>
  <slot>add_user()</slot>
  <slot>export_to()</slot>
 </slots>
</ui>
